app/views/sites/no_results_pages/edit.html.haml

```diff
@@ -13,7 +13,7 @@
   %fieldset
     %p When there are no results for a visitor's search, we display the following default message on the no results page:
     %p
-      %em Sorry, no results found for 'abcxyz'. Try entering fewer or broader query terms.
+      %em Sorry, no results found for 'abcxyz'. Try entering fewer or more general search terms.
```

Comment:
We can use the translation here instead of hard-coding it:
```ruby
%em= t(:no_results_for_and_try, query: 'abcxyz')
```
That will ensure the text is correct if we ever change that translation again.

Reply:
Updated! Re-requesting review.